### DISCUSSION TITLE
Сhange exposed port

### DISCUSSION THREAD

**Discussion by DenisKrym** [wg-easy/wg-easy#892] [Q&A] [answered]
Hi, no matter how much I try to change the setting when running docker, I can't get the Listen Port on the server to change. The Web UI port is changed at the same time. Maybe I don't understand something?

![Вставленное изображение](https://github.com/wg-easy/wg-easy/assets/151864205/71bff0c5-61f3-40be-8862-d747ff13a259)
![Вставленное изображение 1](https://github.com/wg-easy/wg-easy/assets/151864205/d099d0a9-24a2-403e-b6f5-9d5d28c78e5c)
![Вставленное изображение 2](https://github.com/wg-easy/wg-easy/assets/151864205/188348b1-3b8b-49d0-9159-45694ba1ddd8)

**Comment by eandersons** [comment, marked as answer]
The relevant part is this:

```
  -p 51850:51820/udp \
  -p 51851:51821/tcp \
```

Before the colon is the port on the host that the port in container (the number after the colon) should be published (mapped) to.
As it can be seen in the Dockerfile, container exposes ports 51820 and 51821:

https://github.com/wg-easy/wg-easy/blob/db2dc8959b319be4cc891f923856e0e48e585b34/Dockerfile#L43-L44

**Reply by DenisKrym** [reply]
Your advice helped, didn't know docker worked there before, thanks a lot!
